src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

```diff
@@ -271,6 +280,10 @@ private void upsertBuild(Run run, RunExt wfRunExt) {
           nodeLinks.getLog().setHref(joinPaths(rootUrl, nodeLinks.getLog().href));
         }
       }
+      StatusExt status = stage.getStatus();
```

Comment:
@rupalibehera - these changes around pending input:  do they stem from a) the secret -> credential changes, b) the introduction of multi-branch/github folder, c) some other need deemed useful, or c) a bug you found during testing ?

The need for this is not obvious from the code .... a comment somewhere would be good.